ross/fluid_flow/cylindrical.py

```diff
@@ -205,11 +180,70 @@ def __init__(
         self.dz = self.dZ * self.L
         self.dy = self.dY * self.betha_s * self.R
 
-        self.Zdim = self.Z * L
+        self.Zdim = self.Z * self.L
+
+        self.lubricant_dict = {
+            "ISOVG32": {
+                "viscosity1": Q_(4.05640e-06, "reyn").to_base_units().m,
+                "temp1": Q_(40.00000, "degC").to_base_units().m,
+                "viscosity2": Q_(6.76911e-07, "reyn").to_base_units().m,
+                "temp2": Q_(100.00000, "degC").to_base_units().m,
+                "lube_density": Q_(873.99629, "kg/m³").to_base_units().m,
+                "lube_cp": Q_(1948.7995685758851, "J/(kg*degK)").to_base_units().m,
+                "lube_conduct": Q_(0.13126, "W/(m*degC)").to_base_units().m,
+            },
+            "ISOVG46": {
+                "viscosity1": Q_(5.757040938820288e-06, "reyn").to_base_units().m,
+                "temp1": Q_(40, "degC").to_base_units().m,
+                "viscosity2": Q_(8.810775697672788e-07, "reyn").to_base_units().m,
+                "temp2": Q_(100, "degC").to_base_units().m,
+                "lube_density": Q_(862.9, "kg/m³").to_base_units().m,
+                "lube_cp": Q_(1950, "J/(kg*degK)").to_base_units().m,
+                "lube_conduct": Q_(0.15, "W/(m*degC)").to_base_units().m,
+            },
+            "TEST": {
+                "viscosity1": Q_(2.758e-6, "reyn").to_base_units().m,
+                "temp1": Q_(121.7, "degF").to_base_units().m,
+                "viscosity2": Q_(1.119e-6, "reyn").to_base_units().m,
+                "temp2": Q_(175.7, "degF").to_base_units().m,
+                "lube_density": Q_(8.0e-5, "lbf*s²/in⁴").to_base_units().m,
+                "lube_cp": Q_(1.79959e2, "BTU*in/(lbf*s²*degF)").to_base_units().m,
+                "lube_conduct": Q_(2.00621e-6, "BTU/(in*s*degF)").to_base_units().m,
+            },
+        }
+
+        lubricant_properties = self.lubricant_dict[self.lubricant]
+        T_muI = lubricant_properties["temp1"] - 273.15
+        T_muF = lubricant_properties["temp2"] - 273.15
+        mu_I = lubricant_properties["viscosity1"]
+        mu_F = lubricant_properties["viscosity2"]
+        self.rho = lubricant_properties["lube_density"]
+        self.Cp = lubricant_properties["lube_cp"]
+        self.k_t = lubricant_properties["lube_conduct"]
 
         # Interpolation coefficients
         self.a, self.b = self._interpol(T_muI, T_muF, mu_I, mu_F)
 
+        self.run(x0)
+
+        coefs = self.coefficients("lund",False)
+        stiff = True
+        for coef in coefs:
+            if stiff:
+                kxx = coef[0]
+                kxy = coef[1]
+                kyx = coef[2]
+                kyy = coef[3]
+
+                stiff = False
+            else:
+                cxx = coef[0]
+                cxy = coef[1]
+                cyx = coef[2]
+                cyy = coef[3]
+
+        super().__init__(node,kxx,cxx,kyy,kxy,kyx,cyy,cxy,cyx)
```

Comment:
The `super().__init__` call does not have the `frequency` argument here. This way, we can only have a bearing with constant coefficients instantiated. We should have a way to call `THDCylindrical(..., frequency=array`), to calculate the coefficients to different frequencies and pass that to the `super().__init__()` call.